index.html

```diff
@@ -1,19 +1,131 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
+    <title>My Resume</title>
+    <link rel="stylesheet" href="css/index.css" />
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="nav">
+          <ul class="nav-list-container">
+            <li class="list-item">
+              <a href="index.html" class="nav-link">Home</a>
+            </li>
+            <li class="list-item"><a href="#!" class="nav-link">About</a></li>
+            <li class="list-item">
+              <a href="#!" class="nav-link">Portfolio</a>
+            </li>
+            <li class="list-item"><a href="#!" class="nav-link">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <!--/.header-->
+
+    <main class="main-section">
+      <div class="container">
+        <article class="showcase">
+          <section class="dev-intro">
+            <img
+              src="./img/jonny_img.png"
+              alt="johnny image"
+              class="showcase-img"
+            />
+            <h2>Jonny Appleseed-Web Dev</h2>
+            <p>Technology is my jam!</p>
+            <hr />
+            <p>
+              Occasionally I have time to work on side projects, if you are
+              interested, let me know by sign in up on this website
+            </p>
+            <a href="#!">Lets Team Up</a>
+          </section>
+        </article>
+        <!--/.showcase-->
+
+        <section class="dev-details">
+          <article class="skills">
```

Comment:
Don't use too much the `<article>` tag - it had a defined use case. Read more on it https://www.w3schools.com/tags/tag_article.asp

Reply:
Sure, I will read about it.